### DISCUSSION TITLE
Welcome to open-source-python-projects-2023 Discussions!

### DISCUSSION THREAD

**Discussion by mahimairaja** [mahimairaja/open-source-python-projects-2023#1] [Announcements]
## 👋 Welcome to  open-source-python-projects-2023 !

  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kessy-cyber** [comment]
This keeps up in 2023 or we can begin before the year runs out?

**Reply by mahimairaja** [reply]
We are so glad to hear that you are fascinated by open-source projects, and we want you to contribute your best to the repository. The project repository is live now. You can make your pull requests absolutely!
